openworld/sdk/core/client/api.py

```diff
@@ -47,56 +49,61 @@ def __init__(self, config: ClientConfig):
         self.endpoint = config.endpoint
 
     @staticmethod
-    def __build_response(response: requests.Response, response_model):
+    def __build_response(response: requests.Response, response_models: list[pydantic.BaseModel]):
         if response.status_code not in OK_STATUS_CODES_RANGE:
             raise service_exception.OpenWorldServiceException.of(
-                error=Error.from_json(response.json()),
+                error=Error.parse_obj(response.json()),
                 error_code=HTTPStatus(response.status_code),
             )
 
-        if response_model is None:
-            return None
+        response_object = None
+        for model in response_models:
+            if not model:
+                continue
+            try:
+                response_object = pydantic.parse_obj_as(model, response.json())
+                return response_object
+            except Exception:
+                continue
 
-        return response_model.from_dict(response.json())
+        return response_object
 
     def call(
         self,
         method: str,
-        url: str,
-        obj: Any = None,
-        request_headers: Optional[dict] = None,
-        response_model: Optional[Any] = None,
+        url: furl,
+        body: pydantic.BaseModel,
+        headers: dict = dict(), # noqa
```

Comment:
Did you try `{}` (empty dict) instead of `dict()`?

Reply:
I did.